source/protobuf/debugsessionproperties_restricted.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
```

Comment:
Can you check how/if this is included in client drops?

I believe that we wanted the `restricted` protos to NOT be in the main client artifacts. But possibly wanted to create a separate `restricted` client zip artifact. But I'm not sure where that's at or what the mechanism is. And, I expect it would be different for a built in service.

Reply:
@jfriedri-ni is working on creating the restricted client artifact.  Currently, it would appear in the public client artifact.  How do you propose we decorate it to show that it should not be included?  Should I move it to a different directory, protobuf_restricted?  Is there some way we can associate it with the code readiness Restricted value?  Other ideas?

Reply:
That sounds good. I like moving it to a `restricted` sub-directory, or even filtering on `*_restricted.proto`. Associating it with a `CodeReadiness` directly sounds more complicated and isn't necessary for this change. I don't think we need to support the `NextRelease` or `RestrictedNextRelease` granularity.